api/transport/websocket/helpers.go

```diff
@@ -0,0 +1,60 @@
+package websocket
+
+import (
+	"github.com/gorilla/websocket"
+	"github.com/spaceuptech/space-api-go/api/model"
+)
+
+func (s *Socket) setSocket(socket *websocket.Conn) {
+	s.mux.Lock()
+	s.Socket = socket
+	s.mux.Unlock()
+}
+
+func (s *Socket) getConnected() bool {
+	s.mux.RLock()
+	defer s.mux.RUnlock()
+	return s.isConnected
+}
+
+func (s *Socket) setConnected(value bool) {
+	s.mux.Lock()
+	s.isConnected = value
+	s.mux.Unlock()
+}
+
+func (s *Socket) setWriterChannel(ch chan model.WebsocketMessage) {
+	s.mux.Lock()
+	s.SendMessage = ch
+	s.mux.Unlock()
+}
+
+func (s *Socket) RegisterCallback(Type string, function func(data interface{})) {
```

Comment:
All exported functions must have a comment and be in a file named `operations.go`

Reply:
done